core/src/components/content/content.tsx

```diff
@@ -168,12 +169,23 @@ export class Content implements ComponentInterface {
       readTask(ts => {
         this.queued = false;
         this.detail.event = ev;
-        updateScrollDetail(this.detail, this.scrollEl, ts, shouldStart);
+        updateScrollDetail(this.detail, this.scrollEl!, ts, shouldStart);
         this.ionScroll.emit(this.detail);
       });
     }
   }
 
+  // scroll methods can be called in lifecycle hooks where scrollEl isn't ready
+  // (ex: Vue onMounted)
+  // ensure scrollEl is present
+  private async awaitScrollEl() {
```

Comment:
It might make sense to do this in the `getScrollElement` method instead so that those using the `getScrollElement` method get this fix too.

From there, we can do the following in methods like `scrollToBottom`:

```js
const scrollEl = await this.getScrollElement();
```